jsonexporter/config.go

```diff
@@ -6,42 +6,50 @@ import (
 	"io/ioutil"
 )
 
-type Config struct {
-	Name   string            `yaml:name`
-	Path   string            `yaml:path`
-	Labels map[string]string `yaml:labels`
-	Type   string            `yaml:type`
-	Help   string            `yaml:help`
-	Values map[string]string `yaml:values`
+type ModuleConfig struct {
+    Endpoint string            `yaml:"endpoint"`
+    Headers  map[string]string `yaml:"headers"`
+    Mappings []*Mapping        `yaml:"mappings"`
 }
 
-func (config *Config) labelNames() []string {
-	labelNames := make([]string, 0, len(config.Labels))
-	for name := range config.Labels {
+type Mapping struct {
+	Name   string              `yaml:name`
```

Comment:
looks like it has been from the original code but tag values needed to be quoted by `"`?